ci-operator/step-registry/ipi/conf/nutanix/extended/ipi-conf-nutanix-extended-ref.yaml

```diff
@@ -0,0 +1,29 @@
+ref:
+  as: ipi-conf-nutanix-extended
+  from_image:
+    namespace: origin
+    name: centos
+    tag: '8'
+  commands: ipi-conf-nutanix-extended-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: VMS_BOOTTYPE
+    default: "UEFI"
+    documentation: "bootType"
+  - name: CATEGORIES_KEY
+    default: "qe-category-key"
+    documentation: "categories key"
+  - name: CATEGORIES_VALUE
+    default: "qe-category-value"
```

Comment:
The CATEGORIES items need to be created in advance, then save them into cluster profile vault, right?

Reply:
It's similar to labels added to VMS, may not needed to add to valut.

Reply:
ah, got it.